tests/test_wasm_build.py

```
@@ -0,0 +1,290 @@
+        # Help should exit with 0
+        assert result.returncode == 0
+        assert "Usage" in result.stdout or "build" in result.stdout.lower()
+
+
+class TestWasmBuild:
+    """Tests for actual WASM build (only runs if tools are available)."""
+
+    @pytest.fixture
+    def has_build_tools(self):
+        """Check if build tools are available."""
+        emcc_available = subprocess.run(["which", "emcc"], capture_output=True).returncode == 0
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)